configure.ac

```diff
@@ -57,10 +57,40 @@ AC_ARG_WITH([rootlibdir],
         [], [with_rootlibdir=$libdir])
 AC_SUBST([rootlibdir], [$with_rootlibdir])
 
+AC_ARG_WITH([systemdsystemunitdir],
+			AC_HELP_STRING([--with-systemdsystemunitdir=DIR],
+			[path to systemd system unit directory]),
+					[path_systemunitdir=${withval}])
+if (test "${enable_systemd}" != "no" && test -z "${path_systemunitdir}"); then
+	AC_MSG_CHECKING([systemd system unit dir])
+	path_systemunitdir="`$PKG_CONFIG --variable=systemdsystemunitdir systemd`"
```

Comment:
If I run `./configure` on a clean mavlink-router (just after `./autogen.sh`, of course), I get this error

```
checking systemd system unit dir... ./configure: line 17114: --variable=systemdsystemunitdir: command not found
```
(both on ubuntu or arch)
I can only get this working if I disable systemd using `--disable-systemd`. My guess is that `$PKG_CONFIG` is not being set on my environments. Am I missing something?

I'm not well acquainted with autotools, but maybe macro `PKG_CHECK_VAR` can be of help? See https://autotools.io/pkgconfig/variables.html